adr/adr-01-vfs/adr-01-vfs.md

```diff
@@ -0,0 +1,68 @@
+# ADR 1: Chain and VFS Path Referencing
+
+## Changelog
+
+- 7DEC2024: Intial draft
+
+## Status
+
+Proposed
+
+## Context
+
+ADOs will need to be able to reference resources locally and on other chains. It needs to be human readable, standardized, and flexible enough for future expansion as the protocol grows. The primary usage of this schema will be for aOS kernel consumption. Optionally, other protocols can use the schema including offchain environments such as a web browser or local command line.
+
+## Decision
+
+Adopt the already standardized URI ([Uniform Resource Indicator](https://en.wikipedia.org/wiki/Uniform_Resource_Identifier_)). We have borrowed many standard from the POSIX frameworks that have been in use since the 1960s and are ubiquitious across all systems worldwide.
+
+![image](https://upload.wikimedia.org/wikipedia/commons/thumb/d/d6/URI_syntax_diagram.svg/2136px-URI_syntax_diagram.svg.png)
+
+- scheme = andr://
+- userinfo = aOS username
+- host = blockchain (Andromeda Chain being the authority, for now)
+- port = n/a for now (\*)
+- path = vfs handled by host VFS registry
+- query = execution or query parameters
+- fragment = n/a for now (\*)
+
+(\*) These are reserved for future use and declared for backwards compatibility.
+
+Examples:
+
+- andr://osmosis/home/alice/splitter?count
```

Comment:
Given the above example structure would this not be written as `andr://alice@osmosis/home/splitter`?

Reply:
No sir, because the "/" after "osmosis" is the root of the local VFS. You would only send over alice@ if you needed to query something as that user.

Alice's splitter lives here: /home/alice/splitter